### ISSUE TITLE
[custom] pcl-1.12.0-rc1 compiled with wrong version

### ISSUE THREAD

**Issue by whatisor** [PointCloudLibrary/pcl#4970]
this tag compiled with wrong version in cmakelist: project(PCL VERSION 1.11.1.99)

**Comment by kunaltyagi** [comment]
It's a release candidate, not a release.

@mvieth @larshg Thoughts?

**Comment by larshg** [comment]
I don't think we should spent much time on that. Now its documented (by this issue).

**Comment by mvieth** [comment]
@whatisor Would you expect the version in cmake to be 1.12.0 or something else? What is the problem with it being 1.11.1.99?
- The release candidates (e.g. pcl-1.12.0-rc1) are not meant for "normal" use, use the actual releases instead (e.g. pcl-1.12.0)
- A release candidate comes before the release, so it makes sense that the version of the release candidate is less than the version of the release (1.11.1.99 comes before 1.12.0)
- I think it also makes sense that only the actual releases have versions like 1.12.0, 1.11.1, 1.10.0 etc, because only those should be considered by `find_package`, e.g. `find_package(PCL 1.12 REQUIRED)`, not the release candidates.

So overall, I don't see anything wrong?

**Comment by kunaltyagi** [comment]
Agree strongly with the 3rd point raised by @mvieth 

Re: @larshg 

We can do nothing at this point, but this conversation might steer us towards a different direction for the next release (but I quite agree with mvieth, so might be unlikely)
